azkaban-common/src/main/java/io/kubernetes/autoscaling/models/V1beta2VerticalPodAutoscalerStatusRecommendationContainerRecommendations.java

```diff
@@ -0,0 +1,240 @@
+  @SerializedName(SERIALIZED_NAME_LOWER_BOUND)
+  private Map<String, Object> lowerBound = null;
+
+  public static final String SERIALIZED_NAME_TARGET = "target";
+  @SerializedName(SERIALIZED_NAME_TARGET)
+  private Map<String, Object> target = new HashMap<>();
+
+  public static final String SERIALIZED_NAME_UNCAPPED_TARGET = "uncappedTarget";
+  @SerializedName(SERIALIZED_NAME_UNCAPPED_TARGET)
+  private Map<String, Object> uncappedTarget = null;
+
+  public static final String SERIALIZED_NAME_UPPER_BOUND = "upperBound";
+  @SerializedName(SERIALIZED_NAME_UPPER_BOUND)
+  private Map<String, Object> upperBound = null;
+
+
+  public V1beta2VerticalPodAutoscalerStatusRecommendationContainerRecommendations containerName(String containerName) {
+
+    this.containerName = containerName;
+    return this;
+  }
+
+   /**
+   * Name of the container.
+   * @return containerName
+  **/
+  @javax.annotation.Nullable
+  @ApiModelProperty(value = "Name of the container.")
+
+  public String getContainerName() {
+    return containerName;
+  }
+
+
+  public void setContainerName(String containerName) {
+    this.containerName = containerName;
+  }
+
+
+  public V1beta2VerticalPodAutoscalerStatusRecommendationContainerRecommendations lowerBound(Map<String, Object> lowerBound) {
+
+    this.lowerBound = lowerBound;
+    return this;
+  }
+
+  public V1beta2VerticalPodAutoscalerStatusRecommendationContainerRecommendations putLowerBoundItem(String key, Object lowerBoundItem) {
+    if (this.lowerBound == null) {
+      this.lowerBound = new HashMap<>();
+    }
+    this.lowerBound.put(key, lowerBoundItem);
+    return this;
+  }
+
+   /**
+   * Minimum recommended amount of resources. Observes ContainerResourcePolicy. This amount is not guaranteed to be sufficient for the application to operate in a stable way, however running with less resources is likely to have significant impact on performance/availability.
+   * @return lowerBound
+  **/
+  @javax.annotation.Nullable
+  @ApiModelProperty(value = "Minimum recommended amount of resources. Observes ContainerResourcePolicy. This amount is not guaranteed to be sufficient for the application to operate in a stable way, however running with less resources is likely to have significant impact on performance/availability.")
+
+  public Map<String, Object> getLowerBound() {
+    return lowerBound;
+  }
+
+
+  public void setLowerBound(Map<String, Object> lowerBound) {
+    this.lowerBound = lowerBound;
+  }
+
+
+  public V1beta2VerticalPodAutoscalerStatusRecommendationContainerRecommendations target(Map<String, Object> target) {
+
+    this.target = target;
+    return this;
+  }
+
+  public V1beta2VerticalPodAutoscalerStatusRecommendationContainerRecommendations putTargetItem(String key, Object targetItem) {
+    this.target.put(key, targetItem);
+    return this;
+  }
+
+   /**
+   * Recommended amount of resources. Observes ContainerResourcePolicy.
+   * @return target
```

Comment:
Are the comments neccesary if the value off ApiModel is the same as the comment?

Reply:
See the description. The first commit can be ignored. Those VPA client code are auto-generated .